src/scheduler.cc

```diff
@@ -9,23 +9,71 @@
 #include <dirent.h>
 #include <dlfcn.h>
 #include <link.h>
-#include "rccl/rccl.h"
+#include <string>
+#include <curl/curl.h>
+#include <nlohmann/json.hpp>
+
+#ifdef RCCL
+  #include "rccl/rccl.h"
+#else 
+  #include "nccl.h"
+#endif
 #include "parser.h"
 
 #define __hidden __attribute__ ((visibility("hidden")))
 
-#define MSCCL_SCHEDULER_NAME "github.com/microsoft/msccl-scheduler"
+#define MSCCL_SCHEDULER_NAME "github.com/Azure/msccl-scheduler"
 
 static const char* mscclAlgoDirEnv = "MSCCL_ALGO_DIR";
 static const char* mscclAlgoDefaultDir = "msccl-algorithms";
 extern "C" bool mscclUnitTestMode() __attribute__((__weak__));
 static const char* mscclUnitTestAlgoDefaultDir = "msccl-unit-test-algorithms";
-static const char* mscclAlgoShareDirPath = "share/rccl/msccl-algorithms";
-static const char* mscclUnitTestAlgoShareDirPath = "share/rccl/msccl-unit-test-algorithms";
+static const char* mscclAlgoShareDirPath = "share/msccl-scheduler/msccl-algorithms";
```

Comment:
Should be ../share?

Reply:
Ref: https://github.com/ROCmSoftwarePlatform/rccl/blob/develop/src/misc/msccl/msccl_lifecycle.cc#L78-L79

Reply:
changed accordingly.